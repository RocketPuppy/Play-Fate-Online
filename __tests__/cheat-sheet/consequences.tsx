import * as React from "react";
import * as renderer from "react-test-renderer";
import Consequences from "../../src/cheat-sheet/consequences";

test("renders all the pieces", () => {
  const render = renderer.create(<Consequences />);

  expect(render.toJSON()).toMatchSnapshot();
});
