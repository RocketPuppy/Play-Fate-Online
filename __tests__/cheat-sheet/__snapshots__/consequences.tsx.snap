// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Consequences
  </h2>
  <dl>
    <dt>
      Mild
    </dt>
    <dd>
      -2 to attack value
    </dd>
    <dt>
      Moderate
    </dt>
    <dd>
      -4 to attack value
    </dd>
    <dt>
      Severe
    </dt>
    <dd>
      -6 to attack value
    </dd>
    <dt>
      Extreme
    </dt>
    <dd>
      -8 to attack value and permanent character aspect
    </dd>
  </dl>
</div>
`;
